api/src/services/script-runner.service.ts

```diff
@@ -53,6 +54,46 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @param jurisdictionIdDTO id containing the jurisdiction id we are creating the new community type for
+   * @returns successDTO
+   * @description creates a new reserved community type. Reserved community types also need translations added
+   */
+  async createNewReservedCommunityType(
+    req: ExpressRequest,
+    jurisdictionIdDTO: IdDTO,
+  ): Promise<SuccessDTO> {
+    // script runner standard start up
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      'Senior Veterans Community Type',
+      requestingUser,
+    );
```

Comment:
oh thats a good call out, kinda like how I responded in the above new reserved community types would require translations, so we can make modifications then

since this is pretty specific to doorway and this community type I'm not sure if we should make that change here